src/Sylius/Bundle/ApiBundle/Controller/GetStatisticsAction.php

```diff
@@ -90,24 +91,32 @@ public function __invoke(Request $request): Response
     private function createInputDataConstraints(): array
     {
         return [
-            new SymfonyConstraints\Sequentially([
-                new SymfonyConstraints\Collection([
-                    'channelCode' => new Constraints\Code(),
-                    'startDate' => [
-                        new SymfonyConstraints\NotBlank(),
-                        new SymfonyConstraints\DateTime('Y-m-d\TH:i:s', message: 'sylius.date_time.invalid'),
-                    ],
-                    'interval' => new SymfonyConstraints\Choice(choices: array_keys($this->intervalsMap), multiple: false),
-                    'endDate' => [
-                        new SymfonyConstraints\NotBlank(),
-                        new SymfonyConstraints\DateTime('Y-m-d\TH:i:s', message: 'sylius.date_time.invalid'),
-                    ],
-                ]),
-                new SymfonyConstraints\Expression(expression: 'value["startDate"] < value["endDate"]', message: 'sylius.statistics.end_date.invalid'),
+            new SymfonyConstraints\Collection([
+                'channelCode' => new Constraints\Code(),
```

Comment:
I think there is only one ApiBundle constraint, so you can do
```php
use Sylius\Bundle\ApiBundle\Validator\Constraints\Code;
...
use Symfony\Component\Validator\Constraints;
```

```suggestion
                'channelCode' => new Code(),
```